src/components/CardPrimary/CardPrimary.tsx

```diff
@@ -13,6 +13,7 @@ export interface CardPrimaryProps {
   infoUrl?: string;
   infoText?: string;
   size?: "sm" | "md";
+  onButtonClick?: MouseEventHandler<HTMLElement>;
```

Comment:
i think `onButtonClick` is a bit confusing and might make the user think it applies with a button inside the card? it's already possible to add a `onClick` event to the card, shouldn't we use the default porp?